ambari-web/app/controllers/global/cluster_controller.js

```diff
@@ -344,18 +367,25 @@ App.ClusterController = Em.Controller.extend(App.ReloadPopupMixin, {
    * @param callback
    * @returns {?object}
    */
-  loadStackServiceComponents: function (callback) {
-    var callbackObj = {
-      loadStackServiceComponentsSuccess: callback
-    };
-    return App.ajax.send({
+
+  loadStackServiceComponents: function (stacks, callback, stackPosition) {
+    var self = this;
+    App.ajax.send({
       name: 'wizard.service_components',
       data: {
-        stackUrl: App.get('stackVersionURL'),
-        stackVersion: App.get('currentStackVersionNumber')
+        stackUrl: stacks[stackPosition].stackUrl,
+        stackVersion: stacks[stackPosition].stackVersion
```

Comment:
I don't think there's any reason to include stackVersion here. It's not used by the API call separately from the stackUrl.

Reply:
You are right. I didn't take a look at the call itself. Just modified the values.